src/ixdat/config.py

```diff
@@ -0,0 +1,17 @@
+"""Configuration variables including stnadard data directory
```

Comment:
I don't yet see the exact case when this will make a difference, but definitely see the merits of organizing things in classes.
Done.

Reply:
I can see that I maybe didn't do to well explaining what I meant.

Slight detour: Python does not support making _property_ style implementation of variables at the module level (or at least not as well, see https://stackoverflow.com/questions/880530/can-modules-have-properties-the-same-way-that-objects-can). 

This means that if you wanted to do something when a user changes a setting (like flash that new value through to a file or db or really anything else), there is no way to catch that change, since we cannot do as with object properties and just replace them with a `@property` implementation. But if you initially implement the settings as living as properties on an object then the worlds you oyster and you can change setting implementation at will at any time without touching any of the other code.

Reply:
Got it! Thanks. `CFG = Config()` is now implemented